packages/algob/test/internal/core/params/argument-types.ts

```diff
@@ -243,6 +243,11 @@ describe("argumentTypes", () => {
       fsExtra.createFileSync("A");
       fsExtra.chmodSync("A", 0o000); // assign no permission [read-execute-write: 000]
 
+      // check if permission set or not
```

Comment:
Please update the comment to check why we are skipping the test. Also add  a warning that some tests are skipped with a reason why.

Reply:
ok